pkg/bucket/bucket.go

```diff
@@ -67,15 +71,35 @@ func CmdList() *cobra.Command {
 	return cmd
 }
 
-// RunCreate runs a CLI command
+// RunCreate runs a CLI command. The defualt backingstore will be used as the underlying storage for buckets created using the CLI
 func RunCreate(cmd *cobra.Command, args []string) {
 	log := util.Logger()
 	if len(args) != 1 || args[0] == "" {
 		log.Fatalf(`Missing expected arguments: <bucket-name> %s`, cmd.UsageString())
 	}
 	bucketName := args[0]
 	nbClient := system.GetNBClient()
-	err := nbClient.CreateBucketAPI(nb.CreateBucketParams{Name: bucketName})
+
+	bucketClassName :=  options.SystemName + "-default-bucket-class"
```

Comment:
notice that in provisioner.go file there is GetDefaultBucketClass() function, I recommend you to use it if possible or at least do the same functionality

Reply:
@romayalon Unless there's another way to getting the default bucket class name. What is the difference between what he did than using this function? Except for checking the Ready State?
@BarakSason Seems like we missed it, please Add a Ready check like in the original function.

Reply:
@jackyalbo why not reuse the logic if there is no difference?

Reply:
@romayalon In order to call that method, I'll need to move it as well (together with CreateTieringStructure), is that acceptable?

Reply:
Moved GetDefaultBucketClass along with CreateTieringStructure